apps/search-poc/src/hooks/useEntity.ts

```diff
@@ -0,0 +1,20 @@
+import * as React from 'react';
```

Comment:
As I said in another comment, I think this hook is not necessary. All this code can be replaced with:
```tsx
const { data, loading, error } = useNexus(nexus => nexus.httpGet({ path: decodeURIComponent(props.params.selfUrl) }))
```